test/format_duration_default_test.exs

```diff
@@ -1,4 +1,19 @@
 defmodule DurationFormatDefaultTest do
   use ExUnit.Case, async: true
-  doctest Timex.Format.Duration.Formatters.Default
+  use Timex
+
+  alias Timex.Format.Duration.Formatters
+
+  doctest Formatters.Default
+
+  defp format(megaseconds, seconds, microseconds) do
+    Duration.from_erl({megaseconds, seconds, microseconds})
+      |> Formatters.Default.format
+  end
+
+  test "format negative duration" do
```

Comment:
I decided to keep all other tests here to avoid documentation overload with examples